_posts/2015-08-10-type-driven-development.html

```diff
@@ -44,7 +44,7 @@
     My problem was that I didn't know how long it took to handle a message, so the Polling Consumer would have to measure that as well, constantly updating its estimates of how long it takes to handle a message.
   </p>
   <p>
-    This wasn't a harder problem that I could <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solve it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
+    This wasn't a sufficiently hard problem to thwart my <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solving it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
```

Comment:
It may well do. To be honest, I'm simply unable to divine what it is that you were trying to say in the first instance (I retained *hard* in an attempt to retain the intent but suspect changing it to something else (complex?) might serve to further mangle it) and hence I very definitely going out on a limb putting those words in your mouth :)

I have no concrete solution :(